src/EventCalendar.js

```diff
@@ -6,6 +6,8 @@
 
 import $ from 'jquery';
 import moment from 'moment';
+import 'moment/locale/en-au';
+import 'moment/locale/en-gb';
 import 'moment-recur';
 window._ = require('underscore');
 import 'clndr';
```

Comment:
I think this `_` might be nicer if we went for 
```
import _ from 'underscore'

window._ = _;
```

so as not to mix the UMD streams as it were. On a side note it's incredibly lame that `clndr` expects underscore to be global.